source/includes/_ruby.md

```diff
@@ -455,6 +455,16 @@ The following configuration settings are available:
       </td>
       <td>No</td>
     </tr>
+    <tr>
+      <th>collect_remote_ip</th>
+      <td>
+        Indicates whether Scout persists IP-address of user in order to identify her. 
```

Comment:
English feedback:

`Automatically capture end user IP addresses as part of each trace's context.`

Reply:
@cschneid Fixed.